packages/editor/src/dataviews/fields/index.ts

```diff
@@ -7,17 +7,18 @@ import type { Field } from '@wordpress/dataviews';
 /**
  * Internal dependencies
  */
-import type { BasePost, PostWithPageAttributesSupport } from '../types';
+import type { BasePost } from '../types';
+import { getItemTitle } from '../actions/utils';
 
 export const titleField: Field< BasePost > = {
 	type: 'text',
 	id: 'title',
 	label: __( 'Title' ),
 	placeholder: __( 'No title' ),
-	getValue: ( { item } ) => item.title,
+	getValue: ( { item } ) => getItemTitle( item ),
 };
 
-export const orderField: Field< PostWithPageAttributesSupport > = {
+export const orderField: Field< BasePost > = {
```

Comment:
What's the relation between these fields + `.../types` with the `usePostFields` hook defined in edit-site? Last I [tried](https://github.com/WordPress/gutenberg/pull/63849) to consolidate the two in a single place there was a [few dependencies ](https://github.com/WordPress/gutenberg/pull/63849#discussion_r1687873306) in the `usePostFields` hook to get rid of.